<launch>
    <include file="$(find easondrone_gazebo)/launch/mavros_posix_sitl.launch"/>

    <node name="filter_pc" type="filter_pc" pkg="easondrone_mapping"/>

    <include file="$(find easondrone_gazebo)/launch/transform.launch"/>

    <include file="$(find px4ctrl)/launch/px4ctrl.launch"/>

    <node pkg="easondrone_mapping" type="merge_pc2" name="merge_pc2">
        <rosparam command="load" file="$(find easondrone_mapping)/config/merge_pc2.yaml" />
    </node>

    <include file="$(find easondrone_mapping)/launch/octomap_server.launch"/>

    <node type="odom_visualization" pkg="odom_visualization" name="odom_visualization"/>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find easondrone_mapping)/config/easondrone.rviz"/>
</launch>
